### PR TITLE
Update readme to include the latest change: -no_xtb and the sink filter-out task

### PR DESCRIPTION
**Pull request by xiangchenjhu**
Changes on those info in README:
1) add --no_xtb option
2) add filter_in / filter_out result
3)update  pipeline installation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,8 +115,16 @@ To run the pipeline using example data provided in the repository:
 
 This command uses test data to demonstrate the pipeline's functionality, ideal for initial testing and familiarization.
 
+# Result
+
 ## Note:
-The default output (--o) dataframe is stored in Parquet format due to its efficient storage, faster data access, and enhanced support for complex data structures. The Parquet file can be accessed in several ways:
+* The default output (--o) dataframe is stored in Parquet format due to its efficient storage, faster data access, and enhanced support for complex data structures.
```
Comment (amitschang) [inline README.md:121]: I think `--o` should be either (or both) `-o`, `--output`

**Review by amitschang — APPROVED** [review]
Looks good, minor comment

File: README.md (hunk 1 above)
Comment (xiangchenjhu) [reply README.md]: Thanks! updated it.

**Review by amitschang — APPROVED** [review]

**Comment by xiangchenjhu** [comment]
Thanks for review!

**Pull request merged**